packages/rancher-alerting-drivers/charts/Chart.yaml

```diff
@@ -0,0 +1,45 @@
+name: rancher-alerting-drivers
+home: ""
+sources: []
+version: 1.0.1
+description: The manager for third-party webhook receivers used in the Alertmanger
+keywords:
+- monitoring
+- alertmanger
+- webhook
+maintainers:
+- name: Jiaqi Luo
+  email: ""
+  url: https://github.com/jiaqiluo
```

Comment:
CC: @cbron have we used a general Rancher name and url here before? Not opposed to this, but just curious

Reply:
I have it on Monitoring too, but I think we were planning to remove that at some point iirc.

Reply:
Are they required ? If not I would just remove.